src/main/java/javaClasses/homework_4/davit_balabekyan/staff/Developer.java

```diff
@@ -0,0 +1,49 @@
+package javaClasses.homework_4.davit_balabekyan.staff;
+
+import java.util.Objects;
+
+public class Developer extends Staff {
+
+    private String techStack;
+
+    public Developer(String techStack) {
+        this.techStack = techStack;
+    }
+
+    public Developer() {
+    }
+
+    @Override
+    protected void doWork() {
+        System.out.println("Developer's name is " + getName());
+        System.out.println("Developer's workType is " + getTechStack());
+    }
+
+    protected String getTechStack() {
+        return techStack;
+    }
+
+    protected void setTechStack(String techStack) {
+        this.techStack = techStack;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
When you'll change constructor and methods to add ability to add name and last name, please change equals and hashcode too

Reply:
Done)